test/helper.js

```diff
@@ -239,8 +239,10 @@ module.exports.payloadMethod = function (method, t) {
       sget({
         method: upMethod,
         url: 'http://localhost:' + fastify.server.address().port,
-        body: '',
-        headers: { 'Content-Type': 'application/json' },
+        headers: {
```

Comment:
I think you should add another test here instead of update this.

Reply:
In case of `OPTIONS` method the body parsing would be skipped.
May I restore the tests of other methods which I removed because of https://github.com/fastify/fastify/pull/1027#pullrequestreview-135274087?